src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -218,23 +672,28 @@ def map_featureflag_to_keyvalue(featureflag):
         raise ValueError(error_msg)
 
     except Exception as exception:
-        error_msg = "Exception while converting feature flag to key value: {0}\n{1}".format(featureflag.key, exception)
+        error_msg = (
+            "Exception while converting feature flag to key value: {0}\n{1}".format(
+                featureflag.key, exception
+            )
+        )
         raise Exception(error_msg)  # pylint: disable=broad-exception-raised
 
     return set_kv
 
 
 def map_keyvalue_to_featureflag(keyvalue, show_conditions=True):
-    '''
-        Helper Function to convert KeyValue object to FeatureFlag object for display
+    """
+    Helper Function to convert KeyValue object to FeatureFlag object for display
 
-        Args:
-            keyvalue - KeyValue object to be converted
-            show_conditions - Boolean for controlling whether we want to display "Conditions" or not
+    Args:
+        keyvalue - KeyValue object to be converted
+        show_conditions - Boolean for controlling whether we want to display "Conditions" or not
```

Comment:
Does the show_conditions flag control variants/allocation/telemetry too?

Reply:
Currently it just controls the conditions. I am wondering if we should hide the other new fields. If we want to hide the new fields maybe we should rename the argument to something else other than `show_conditions`?

Reply:
Yes, we should hide the details of other fields too. Is this user configurable? Maybe we can use something like `show_all_details`